hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -647,6 +647,68 @@ public void testDeleteToTrashOrSkipTrash() throws Exception {
     }
   }
 
+  @Test
```

Comment:
Can this test be placed in the integration test suite, `TestRootedOzoneFileSystem`?  Grouped with symlink bucket tests as,
https://github.com/apache/ozone/blob/0c84ff1cdb5208e4932bcc6195b26fcaaeece22b/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java#L1520

Reply:
@neils-dev Cluster initialisation needs to be done with specific OM ID, and need some shell execute method to added again TestRootedOzoneFileSystem, and since this class is not doing shell related testing, so added in TestOzoneShellHA,
I think this is not required as way of testing the link bucket is different in both case.